podcast/2.10.md

```diff
@@ -0,0 +1,778 @@
+
+But there's a lot of other projects that need to interface with the external world. Android has lots of partners and so Android partners have to download the Android source code and of course they can't use Piper for that because Piper's an internal product.
+
+The Chrome web browser is something that people outside of Google also want to develop on and so they need to download the source code and they need to do code review and so they need external tools for that and so they use Git.
+
+And to come back to Code Search, so there's Code Search for Google 3, which is a standardized platform, and so you can do things like not just index the source code but you can, because it's the standard platform, you can also run a compiler and then the compiler tells you where the symbols are and you can do cross-referencing.
+
+**Beyang Liu:**
+
+Yeah.
+
+**Han-Wen Nienhuys:**
+
+Anyway, Shawn wanted to have Code Search for people that were using Gerrit and the Code Search team at the time was in Munich and he said, "Well why don't you go chat with these folks and see what you can come up with" and so I did.
+
+One of the people that were involved explained to me how to build code search, what algorithms you do, and then he also said to me that that's a lot of work to reimplement from scratch. Maybe we should just open source our internal Code Search, which is written in C++, and it will be much faster than if you write it in Go, as it turns out.
+
+If you say to me, "Han-Wen, this is way too difficult. You're not going to be able to do that." That is waving a red flag to a bull and that's why I thought, I'm going to prove you wrong. I'm going to prove that this is actually really easy.
+
+So I implemented the first version of this search thing, and if you go back in the history of Zoekt, you can still find it. And it's actually fairly simple, so the basic idea is very simple and I got it to work in, I don't know, it took me one or two weeks.
+
+But then you see it working and it's super fast, and it's an interesting problem, and addictive to work on and so I immediately saw all these ways in which I could do it–make it more fancy, faster.
+
+And so I've been working on it. It started really small and then I thought, well, I should do regular expressions and then I should do indexing Git repositories and then maybe I should do multiple branches, and then...
+
+And so every time, and I was doing it on the side and it was a fun project, and so I was doing my normal job but in spare moments where my mind could wander off, I was always thinking about what the next step was for making my code search thing even more amazing.
+
+**Beyang Liu:**
+
+Yeah.
+
+**Han-Wen Nienhuys:**
+
+So I got it to a point that it was really working quite well. I got my friends from the Bazel team to host me an instance. There's a cs called bazel.build and if you go there, there's an instance running Zoekt, and I use that fairly regularly to search code.
```

Comment:
What does "cs" refer to here?

Reply:
@csbailey5t my searches turned up this https://cs.bazel.build/ which looks like what he was referring to. Doesn't clarify for me what "cs" is, though. Could we link to it, maybe?

Reply:
@nickmyyz I listened to the recording, and he's saying that there's an instance, cs.bazel.build - https://cs.bazel.build/